WordPress/Classes/ViewRelated/Post/PrivateSiteURLProtocol.m

```diff
@@ -21,7 +21,9 @@ + (void)registerPrivateSiteURLProtocol
 {    
     @synchronized(mutex) {
         if (regcount == 0) {
-            [NSURLProtocol registerClass:[self class]];
+            if (![NSURLProtocol registerClass:[self class]]) {
+                DDLogInfo(@"Unable to register protocol");
```

Comment:
We should consider an assertion before the log (so that we get a loud warning on debug builds).